family_foto/web.py

```diff
@@ -72,6 +73,9 @@ def upload():
     """
     if 'file' in request.files:
         file: FileStorage = request.files['file']
+        exists: File = File.query.filter_by(filename=file.filename).first()
```

Comment:
Is the file name really enough?
What if you want to upload two files that are called Unnamed.jpeg ?
They may be different!

Reply:
Yes, you are right. I thought for a first check it might be sufficient as most images will be uploaded from a some photographing device with uses timestamps for file names. But I will to implement a more sophisticated check.

Reply:
After checking for the name, I check for the byte content.